Update Set-MpPreference.md (#2560)
MicrosoftDocs/windows-powershell-docs: docset/winserver2019-ps/defender/Set-MpPreference.md

### -DisableScanningNetworkFiles
Indicates whether to scan for network files. If you specify a value of $False or do not specify a value, Windows Defender scans network files. If you specify a value of $True, Windows Defender does not scan network files. We do not recommend that you scan network files.

> Note: This parameter is specific for Quick and Scheduled scans.

Reviewer: ```suggestion
Note: This parameter applies specifically to quick and scheduled scans.
```
